mixer/pkg/attribute/mutableBag.go

```diff
@@ -277,8 +277,9 @@ func (mb *MutableBag) ToProto(output *mixerpb.CompressedAttributes, globalDict m
 				output.StringMaps = make(map[int32]mixerpb.StringMap)
 			}
 			output.StringMaps[index] = mixerpb.StringMap{Entries: sm}
+
 		default:
-			scope.Errorf("Cannot convert value:%v of type:%T", v, v)
+			panic(fmt.Errorf("cannot convert value:%v of type:%T", v, v))
```

Comment:
Is this path covered by a unit test? pkg/attribute is at 100% coverage, we should keep it that way if possible.

Reply:
No it is not covered. Added a test and codecov is 100% now.